qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -47,15 +50,35 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
 
     private SplunkDeployment splunkDeployment
 
+    @Shared
+    private Policy processTargettingKubelet
+    @Shared
+    private Policy unauthorizedNetworkFlow
+
     def setupSpec() {
         orchestrator.deleteNamespace(TEST_NAMESPACE)
 
         orchestrator.ensureNamespaceExists(TEST_NAMESPACE)
         addStackroxImagePullSecret(TEST_NAMESPACE)
+
+        processTargettingKubelet = Services.clonePolicyAndScopeByTestNamespace(
```

Comment:
Not sure if we want to create such specific method to use it only twice

Reply:
This is a pattern than I expect to use in the next two test specs I plan to parallelize. i.e. cloning a policy and reducing its scope so that it is only triggered by the test spec.